l10n_ch_zip/__init__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please remove this line in all files

In Python 3 the file is by default in utf-8